examples/reference/widgets/CheckBoxGroup.ipynb

```diff
@@ -33,10 +33,12 @@
     "##### Display\n",
```

Comment:
The `CheckBoxGroup` looks like a `RadioBoxGroup`

![image](https://github.com/user-attachments/assets/b222d651-9282-44e7-b10b-6211f3511e3b)

```python
import panel as pn
import panel_material_ui as pmui

pn.extension()

checkbox_group = pmui.CheckBoxGroup(
    label='Checkbox Group', value=['Apple', 'Pear'], options=['Apple', 'Banana', 'Pear', 'Strawberry'],
    inline=True)

checkbox_group.servable()

checkbox_group = pn.widgets.CheckBoxGroup(
    name='Checkbox Group', value=['Apple', 'Pear'], options=['Apple', 'Banana', 'Pear', 'Strawberry'],
    inline=True)

checkbox_group.servable()
```

Reply:
I had honestly never realized that they looked different in Panel!

Reply:
Its in the name - CheckBox 😉